rollup/private/rollup.bzl

```diff
@@ -215,19 +215,15 @@ def _impl(ctx):
             runfiles = runfiles,
         ),
         js_info(
-            npm_linked_package_files = npm_linked_packages.direct_files,
-            npm_linked_packages = npm_linked_packages.direct,
-            npm_package_store_deps = npm_package_store_deps,
+            target = ctx.label,
             sources = output_sources_depset,
-            # Since we're bundling, we don't propogate linked npm packages from dependencies since
-            # they are bundled and the dependencies are dropped. If a subset of linked npm
-            # dependencies are not bundled it is up the the user to re-specify these in `data` if
-            # they are runtime dependencies to progagate to binary rules or `srcs` if they are to be
-            # propagated to downstream build targets.
-            transitive_npm_linked_package_files = npm_linked_packages.direct_files,
-            transitive_npm_linked_packages = npm_linked_packages.direct,
-            # Since we're bundling, we don't propogate any transitive sources from dependencies
+            types = depset(),  # rollup does not emit types directly
```

Comment:
Are these not optional and can be left unset?

Reply:
They are optional and can be left unset but I added them for completeness so there is no ambiguity. Commented on why `types` is an empty depset as well.